server/adaptors/integrations/__data__/repository/apache/schemas/communication-1.0.0.mapping.json

```diff
@@ -0,0 +1,124 @@
+{
+    "template": {
+        "mappings": {
+            "_meta": {
+                "version": "1.0.0",
+                "catalog": "observability",
+                "type": "logs",
+                "component": "communication"
+            },
+            "properties": {
+                "communication": {
+                    "properties": {
+                        "sock.family": {
+                            "type": "keyword",
+                            "ignore_above": 256
+                        },
+                        "source": {
+                            "type": "object",
+                            "properties": {
+                                "address": {
+                                    "type": "text",
+                                    "fields": {
+                                        "keyword": {
+                                            "type": "keyword",
+                                            "ignore_above": 1024
+                                        }
+                                    }
+                                },
+                                "domain": {
+                                    "type": "text",
+                                    "fields": {
+                                        "keyword": {
+                                            "type": "keyword",
+                                            "ignore_above": 1024
+                                        }
+                                    }
+                                },
+                                "bytes": {
+                                    "type": "long"
+                                },
+                                "ip": {
+                                    "type": "ip"
+                                },
+                                "port": {
+                                    "type": "long"
+                                },
+                                "mac": {
+                                    "type": "keyword",
+                                    "ignore_above": 1024
+                                },
+                                "packets": {
+                                    "type": "long"
+                                },
+                                "geo.city_name": {
+                                    "type": "keyword"
+                                },
+                                "geo.country_iso_code": {
```

Comment:
Opposite problem as with the user agent, `geo` should be an object instead of a segment of multiple fields.